examples/hotel-bookings/src/components/hotelData.js

```diff
@@ -0,0 +1,27 @@
+import * as d3 from "npm:d3";
+import {FileAttachment} from "npm:@observablehq/stdlib";
+
+const hotels = await FileAttachment("../data/hotels.csv").csv({typed: true});
+
+export const hotelData = hotels.map((d) => ({
+  ...d,
+  IsCanceled: d.IsCanceled == 0 ? "Keep" : "Cancel",
+  season: ["June", "July", "August"].includes(d.ArrivalDateMonth)
+    ? "Summer"
+    : ["September", "October", "November"].includes(d.ArrivalDateMonth)
+    ? "Fall"
+    : ["December", "January", "February"].includes(d.ArrivalDateMonth)
+    ? "Winter"
+    : "Spring",
+  Country: d.Country == "NULL" ? "Unknown" : d.Country,
+  Meal: d.Meal.trim(),
+  ReservedRoomType: d.ReservedRoomType.trim(),
+  AssignedRoomType: d.AssignedRoomType.trim(),
```

Comment:
Rather than a "component" that loads and transforms the data, the "wrangling/normalization" feels like it should be part of a data loader — or, absent a data loader, we should normalize up front in the csv file, and only use "revive" to parse dates back from ISO to JavaScript.

Reply:
Moved to hotelData.csv.js